orttraining/orttraining/core/optimizer/graph_transformer_utils.cc

```diff
@@ -157,8 +157,10 @@ std::vector<std::unique_ptr<GraphTransformer>> GeneratePreTrainingTransformers(
         transformers.emplace_back(std::make_unique<GeluApproximation>(compatible_eps));
       }
       InlinedHashSet<std::string> excluded_initializers(weights_to_train.begin(), weights_to_train.end());
+      static ConfigOptions empty_config_options;
```

Comment:
nit: could we make all the static empty ConfigOptions variables const? just to provide more assurance that they'll stay empty.